tools/amd_build/patches/a_aten_src_THC_THCTensorMathReduce.cuh.patch

```diff
@@ -0,0 +1,69 @@
+diff --git a/aten/src/THC/THCTensorMathReduce.cuh b/aten/src/THC/THCTensorMathReduce.cuh
+index c2ce92898..b2208c010 100644
+--- a/aten/src/THC/THCTensorMathReduce.cuh
++++ b/aten/src/THC/THCTensorMathReduce.cuh
+@@ -105,7 +105,7 @@ struct SquareFunctor<ResT, half> {
+ template <typename T>
+ struct ReduceMin {
+   inline __device__ T operator()(T a, T b) const {
+-    return (THCNumerics<T>::lt(a, b) ||
++    return ((int)THCNumerics<T>::sub(a, b) < 0 ||
```

Comment:
This int cast looks very questionable

Reply:
@wsttiger can explain further on this. There are issues with the compiler that we needed to work around.